spaces/S000168/properties/P000089.md

```diff
@@ -0,0 +1,12 @@
+---
+space: S000168
+property: P000089
+value: true
+refs:
+  - mathse: 406851
+    name: fixed point of a continuous map on a projective space
+  - mathse: 3672897
+    name: Projective plane, fixed point
+---
+
+See {{mathse:406851}} and {{mathse:3672897}}.
```

Comment:
These two posts have the loose end that they assume the reader is familiar with the fact that the identity and antipodal map of $S^2$ are not homotopic. Should we fill in that gap with another reference? The way I know involves degree theory. For that we could cite Corollary 6.12. of Bredon's book *Topology and Geometry*.

Reply:
Maybe hatcher {{zb:1044.55001}} is enough, Section 2.2.